### PR TITLE
Feat/feature support scale specified

### PR DESCRIPTION
**Pull request by xuefei1313**
[[中文版模板 / Chinese template](https://github.com/VisActor/VChart/blob/main/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)]

### 🤔 This is a ...

- [x] New feature
- [ ] Bug fix
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Refactoring
- [ ] Update dependency
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Release
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Workflow
- [ ] Other (about what?)

### 🐞 Bugserver case id

64b8d52edc862dee5aa01400

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |           |
| 🇨🇳 Chinese |           |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [ ] Doc is updated/provided or not needed
- [ ] Demo is updated/provided or not needed
- [ ] TypeScript definition is updated/provided or not needed
- [ ] Changelog is provided or not needed

---

### 🚀 Summary

copilot:summary

### 🔍 Walkthrough

copilot:walkthrough

### CODE REVIEW

**Comment by kkxxkk2019** [comment]
填一下 关联的 PR

File: packages/vchart/src/typings/visual.ts [hunk 1]
```diff
@@ -14,6 +14,7 @@ export interface IVisualSpecBase<D, T> {
   type: ScaleType;
   domain: D[];
   range: T[];
+  specified?: { [key: string]: unknown };
```
Comment (kkxxkk2019) [inline packages/vchart/src/typings/visual.ts:17]: 这些属性的注释都补充下吧

File: packages/vchart/src/chart/base-chart.ts [hunk 2]
```diff
@@ -726,10 +726,11 @@ export class BaseChart extends CompilableBase implements IChart {
         if (isArray(colorSpec)) {
           colorScaleSpec.range = colorSpec.map(color => getActualColor(color, this._theme?.colorScheme));
         } else {
-          colorScaleSpec.type = (colorSpec as IVisualSpecScale<any, any>).type;
-          (colorSpec as IVisualSpecScale<any, any>).domain &&
-            (colorScaleSpec.domain = (colorSpec as IVisualSpecScale<any, any>).domain);
-          colorScaleSpec.range = (colorSpec as IVisualSpecScale<any, any>).range;
+          const tempSpec = colorSpec as IVisualSpecScale<any, any>;
+          tempSpec.type && (colorScaleSpec.type = tempSpec.type);
```
Comment (kkxxkk2019) [inline packages/vchart/src/chart/base-chart.ts:730]: 是不是用 Object.prototype.hasOwnProperty 会更好一些

Comment (xuefei1313) [reply packages/vchart/src/chart/base-chart.ts:730]: 👌

**Review by kkxxkk2019 — APPROVED** [review]

**Pull request merged**